pkg/acsengine/const.go

```diff
@@ -87,7 +87,7 @@ var KubeImages = map[string]map[string]string{
 		"dashboard":       "kubernetes-dashboard-amd64:v1.6.1",
 		"exechealthz":     "exechealthz-amd64:1.2",
 		"addonresizer":    "addon-resizer:1.7",
-		"heapster":        "heapster:v1.4.0",
+		"heapster":        "heapster-amd64:v1.4.0",
```

Comment:
Sure, I did for `heapster-amd64:v1.4.0` and `heapster-amd64:v1.3.0` but left `heapster:v1.2.0` unchanged since there is no amd64 available. This is also the kubernetes patters.